model_utils/tests/models.py

```diff
@@ -4,25 +4,24 @@
 from django.utils.encoding import python_2_unicode_compatible
 from django.utils.translation import ugettext_lazy as _
 
+from model_utils import Choices
+from model_utils.fields import SplitField, MonitorField, StatusField
+from model_utils.managers import QueryManager, InheritanceManager
 from model_utils.models import (
     SoftDeletableModel,
     StatusModel,
     TimeFramedModel,
     TimeStampedModel,
 )
-from model_utils.tracker import FieldTracker, ModelTracker
-from model_utils.managers import QueryManager, InheritanceManager
-from model_utils.fields import SplitField, MonitorField, StatusField
 from model_utils.tests.fields import MutableField
-from model_utils import Choices
-
+from model_utils.tests.managers import CustomSoftDeleteManager
+from model_utils.tracker import FieldTracker, ModelTracker
 
 
 class InheritanceManagerTestRelated(models.Model):
     pass
 
 
-
```

Comment:
OK... Thanks for your comment. That's the result of "reformat code" in PyCharm (my editor) which by default, I think, follows PEP8.

I don't mind reverting these blank line changes, it's just a matter of being consistent over the whole project I guess (the `managers.py` IS following PEP8 and has 2 lines between classes).

Probably a separate discussion, but surely there must be an automated, customisable way of ensuring that, rather than relying on humans. Something that would be picked up by automated tests and ideally most editors.